code-challenges.test.js

```diff
@@ -21,11 +21,54 @@
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
-
+//PSEUDOCODE
+//This function take the number and  finds the fiboncci numbers ,
+// creates an array of the lengt number and adds them to array and return the array
+
+function fibonacci(number)
+{
+    //initialize
+    var fibonacciNumbers = [];
+    //initalize the fibbonacci increments values
+		let n1 = 1, n2 = 1, nextTerm;
+
+
+// /looop through the range
+for (let i = 1; i <= number; i++) {
+    //push the number to array
+     fibonacciNumbers .push(n1)
+//next fiboncci base start
+    nextTerm = n1 + n2;
+    // increment n1 to  n2 i.e + 1
+    n1 = n2;
+    // increment n2 to nextTerm i.e +1
+    n2 = nextTerm;
+
+}
+    //retunr the array in  json to match the other from test beacuse of serialzation problem
+    return JSON.stringify(fibonacciNumbers);
```

Comment:
this is good stuff but why are you using JSON.stringify? This should not be a json object